website/events/templates/events/admin/details.html

```diff
@@ -55,6 +55,7 @@ <h2>{% trans "cancellations"|capfirst %}</h2>
         <a href="{% url 'admin:events_event_export' pk=event.pk %}" class="button">{% trans "Export registrations" %}</a>
         <a href="{% url 'admin:events_event_export_email' pk=event.pk %}" class="button">{% trans "Export email addresses of attendees" %}</a>
         <a href="{% url 'admin:events_event_message' pk=event.pk %}" class="button">{% trans "Send pushnotification to registered users" %}</a>
+        <a href="mailto:?bcc={% for p in event.participants %}{% if p.member %}{{p.member.email}},{%endif%}{%endfor%}" class="button"> {% trans "Send email to attendees" %} </a>
```

Comment:
Would it not be best to also remove the export functionality in favour of this?

Reply:
From what I can find this seems to be fine, also for very large links, except for Internet Explorer, and possibly opening it in Outlook. But in that case you can still export to csv or open in a different browser.
https://stackoverflow.com/questions/13317429/mailto-max-length-of-each-internet-browsers